site/docs/master/support-process.md

```diff
@@ -0,0 +1,40 @@
+# Support Process
+
+## Weekly Rotation
+
+The Velero maintainers use a weekly rotation to manage community support. Each week, a different maintainer is the point person for responding to incoming support issues via Slack, GitHub, and the Google group. The point person is *not* expected to be on-call 24x7. Instead, they choose one or more hour(s) per day to be available/responding to incoming issues and communicate that to the community.
+
+## Start of Week
+
+Update the public Slack channel's topic to indicate that you are the point person for the week, and what hours you'll be available.
+
+## During the Week
+
+### Where to monitor
+- `#velero` public Slack channel in Kubernetes org
+- [all Velero-related repos][0] in GitHub (`velero`, `velero-plugin-for-[aws|gcp|microsoft-azure|csi]`)
+- [Project Velero Google Group][1]
+
+### GitHub issue flow
+
+Generally speaking, new GitHub issues will fall into one of several categories. Use the following process for each:
+
+1. **Feature request**
+    - Label the issue with `Enhancement/User` or `Enhancement/Dev`
+    - Leave the issue in the `New Issues` swimlane for triage by product mgmt
+1. **Bug**
+    - Label the issue with `Bug`
+    - Leave the issue in the `New Issues` swimlane for triage by product mgmt
+1. **User question/problem** that does not clearly fall into one of the previous categories
+    - When you start investigating/responding, label the issue with `Investigating`
+    - Add comments as you go, so both the user and future support people have as much context as possible
```

Comment:
Maybe add:

- Add the label `Needs Info` whenever the issue is waiting for more information from the user. Remove/re-add the label as needed.